runhouse/rns/hardware/cluster_factory.py

```diff
@@ -37,7 +38,8 @@ def cluster(
         ips (List[str], optional): List of IP addresses for the BYO cluster.
         ssh_creds (dict, optional): Dictionary mapping SSH credentials.
             Example: ``ssh_creds={'ssh_user': '...', 'ssh_private_key':'<path_to_key>'}``
-        dryrun:
+        dryrun (bool): Whether or not to save the cluster. (Default: ``False``)
```

Comment:
I know this is copied from blob but the wording is a bit confusing and I think mixes up the behavior for dryrun, could you make it clearer for both?